data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -34,7 +34,7 @@ done
 
 echo "Gathering bootstrap journals ..."
 mkdir -p "${ARTIFACTS}/bootstrap/journals"
-for service in approve-csr bootkube crio crio-configure ironic kubelet master-bmh-update release-image release-image-download sssd
+for service in approve-csr bootkube crio crio-configure ironic kubelet master-bmh-update release-image release-image-download sssd bootstrap-verify-api-server-urls
```

Comment:
I don't think we should add it here. 
https://github.com/openshift/installer/pull/5816/files#r877383135

Reply:
> I don't think we should add it here. https://github.com/openshift/installer/pull/5816/files#r877383135

If we don't add the service here, would `analyze` still work?

Reply:
I'm not sure what journalctl will pick up here if we add it since we dont create a new file/

Reply:
IIUC logs generated by check_api_url and check_api_int_url would be copied here: https://github.com/openshift/installer/blob/master/data/data/bootstrap/files/usr/local/bin/installer-gather.sh#L20-L22